packages/devtools_app/lib/src/inspector/inspector_tree.dart

```diff
@@ -701,4 +701,4 @@ mixin InspectorTreeFixedRowHeightController on InspectorTreeController {
     targetRect = targetRect.inflate(20.0);
     scrollToRect(targetRect);
   }
-}
+}
```

Comment:
add back new line at end of file - formatter should take care of that

Reply:
fixed